src/components/home/Hero/Title.js

```diff
@@ -3,21 +3,35 @@ import PropTypes from "prop-types"
 
 import PageTitle from "../../PageTitle"
 import Planet from "../../Planet"
+import useDetectJavascript from "../../../common/useDetectJavascript"
 
 import "./Title.module.css"
 
-const Title = ({ planetMorph }) => (
-  <PageTitle>
-    We nurture{" "}
-    <span styleName="ideas">
-      ideas
-      <span styleName="planet">
-        <Planet morph={planetMorph} codeName="heroTittle" color="blue" />
-      </span>
-    </span>{" "}
-    <span styleName="glue">that empower</span> people
-  </PageTitle>
-)
+const Title = ({ planetMorph }) => {
+  const hasJavascript = useDetectJavascript()
+
+  if (hasJavascript) {
```

Comment:
Can you change this to a guard clause? Less braces involved :stuck_out_tongue:

Reply:
Of course :smile:

Reply:
But wait, this is already a guard clause no? I have no second `else`, I only use braces here because of the AirBnB style guide, which enforces braces on multi-line conditionals (I am used to that because we use it on every JS project). 

I can remove them if you really want to though :stuck_out_tongue:

Reply:
Oh yeah, but we still need to return some JSX in both cases, but we can "exit early" with the bad clause, that is when we don't JS available. I'll change it